main.go

```diff
@@ -232,6 +232,10 @@ func readCertsFromFile(wg *sync.WaitGroup, reader io.Reader, filename string, fo
 		scanner := pemScanner(reader)
 		for scanner.Scan() {
 			block, _ := pem.Decode(scanner.Bytes())
+			if block.Type != "CERTIFICATE" {
+				// Skip non-certificate PEM blocks
+				continue
+			}
 			cert, err := x509.ParseCertificate(block.Bytes)
```

Comment:
Shrug - I was just thinking about a case where you might have something that looks like a PEM certificate but isn't really, and the data you're interested in is below it.  I don't have strong feelings either.